org.goldcoin.goldcoin-qt.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.goldcoin.goldcoin-qt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
```

Comment:
so why not build from source?

Reply:
I provide already built binary. I'm sure your system is working properly and provides many options to users but i prefer to use my own building envoriment.